cime_config/config_tests.xml

```diff
@@ -260,6 +260,16 @@ LII    CLM initial condition interpolation test
     <DOUT_S>FALSE</DOUT_S>
   </test>
 
+  <test NAME="PRE">
+    <DESC>pause/resume test, default 5 hours, five pause/resume cycles, no data modification</DESC>
+    <INFO_DBUG>1</INFO_DBUG>
+    <STOP_OPTION>nhours</STOP_OPTION>
+    <STOP_N>5</STOP_N>
+    <HIST_OPTION>nhours</HIST_OPTION>
+    <HIST_N>1</HIST_N>
+    <DOUT_S>FALSE</DOUT_S>
+  </test>
+
```

Comment:
This test is not currently tested (neither is the newish DAE test). I propose adding these to cime_developer in update_acme_tests.py:
DAE.f19_f19.A
PRE.f19_f19_ADESP
PRE.f19_f19.ADESP_TEST

If that is too many, I can leave off the last one.